TestApp/Navigation/Sources/Navigation/Deeplinks/DeeplinkHandler.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+public struct DeeplinkHandlerPreferences {
+
+    enum DestinationTab {
+        case basedOnHandler
+        case custom(TabBarTab)
+    }
+
+    let destinationTab: DestinationTab?
+    let shouldClearCurrentUI: Bool
```

Comment:
Naming suggestion: `dismissAllTabsContent` - or, if this does what I think it does.

Reply:
It dismisses everything from the target tab.

`dismissAllTabsContent` doesn't have any prefix indicating the property is a `Bool`...